### PR TITLE
[BPK-1960] Fix card bugs

### PR DESCRIPTION
**Pull request by georgegillams**

### CODE REVIEW

File: Backpack/Card/Classes/BPKCard.m [hunk 1]
```diff
@@ -28,6 +28,7 @@
 @interface BPKCard()
 
 - (void)setupWithPadded:(BOOL)padded;
+@property(nonatomic) UIView *innerView;
```
Comment (k0nserv) [inline Backpack/Card/Classes/BPKCard.m]: Can we specify `strong` and call this contentView?

File: Backpack/Card/Classes/BPKCard.m [hunk 2]
```diff
@@ -65,9 +66,9 @@ - (instancetype)initWithPadded:(BOOL)padded {
 
 - (void)setPadded:(BOOL)padded {
     if (padded) {
-        self.layoutMargins = UIEdgeInsetsMake(BPKSpacingBase, BPKSpacingBase, BPKSpacingBase, BPKSpacingBase);
+        self.innerView.layoutMargins = UIEdgeInsetsMake(BPKSpacingBase, BPKSpacingBase, BPKSpacingBase, BPKSpacingBase);
```
Comment (k0nserv) [inline Backpack/Card/Classes/BPKCard.m]: I would execpt us to still set the `layoutMaring` on `self` here and have the innerView constrianted to the layout guides of `self`

File: Backpack/Card/Classes/BPKCard.h [hunk 3]
```diff
@@ -32,6 +32,11 @@ NS_SWIFT_NAME(Card) IB_DESIGNABLE @interface BPKCard: UIControl
  */
 @property (nonatomic, assign, getter=isPadded) IBInspectable BOOL padded;
 
+/**
+ * The view to display within the card.
+ */
+@property(nonatomic, assign, nullable) UIView *subview;
```
Comment (tevelee) [inline Backpack/Card/Classes/BPKCard.h]: I was thinking, `contentView` might be a better name. Would align more with UIKit standard naming, for example on UICollectionViewCell.h:
```objc
@property (nonatomic, readonly) UIView *contentView; // add custom subviews to the cell's contentView
```

Comment (k0nserv) [reply Backpack/Card/Classes/BPKCard.h]: Possibly, it needs to make sense for the divided card too

**Pull request merged**